x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/doc_count_chart/doc_count_chart.tsx

```diff
@@ -43,6 +43,7 @@ export const DocCountChart: FC<{
 
   const [eventRateChartData, setEventRateChartData] = useState<LineChartPoint[]>([]);
   const [timeRange, setTimeRange] = useState<{ start: Moment; end: Moment } | undefined>(undefined);
+  const [dataReady, setDataReady] = useState(false);
```

Comment:
checked the code in the file and note sure if we need a dedicated state here: 
```suggestion
  const dataReady = useMemo(() => eventRateChartData.some((d) => d.value > 0), [eventRateChartData])
```

Reply:
I don't want to run this `some` once the data is ready. It's only run when `dataReady` is `false`.
For stateful es, `dataReady` will be `true` almost instantly.